diagnostic.md

```diff
@@ -6,70 +6,89 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+
+
+get '/books/:id', to: 'books#show'
+
+Or
+
+get '/books/:id' => 'books#show'
+
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+in the routes.rb file
+
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+Bundle exec rake is a bundler command that will excecute the script db:migrate and when db:migrate migrates the data in the database.
```

Comment:
`bundle exec rake db:migrate` executes all of the database migration files in `db/migrate` and generates a new database schema file, `db/schema.rb`.